app/controllers/api/v1/worksheets_controller.rb

```diff
@@ -20,9 +20,11 @@ def update
     image_url = remote_image_url
     prepped_params = worksheet_params.except(:image_url, :photo).merge({ image_url: image_url })
     if @worksheet.update(prepped_params)
-      students = @worksheet.work_group.students.to_a
-      teacher = @worksheet.worksheet_template.user
-      DemoMailer.with(students: students, teacher: teacher, image_url: image_url).send_worksheets.deliver_later
+      students = @worksheet.work_group.students.pluck(:email)
+      student_group = @worksheet.work_group
```

Comment:
I added student_group so that in the mailer, I could include the class name into the subject header.